perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -380,6 +380,52 @@ public int compare(Group groupToCompare,Group groupToCompareWith) {
 			}
 		}
 
+		// remove admin roles of group
+		List<Facility> facilitiesWhereGroupIsAdmin = getGroupsManagerImpl().getFacilitiesWhereGroupIsAdmin(sess, group);
+		for (Facility facility : facilitiesWhereGroupIsAdmin) {
+			try {
+				perunBl.getFacilitiesManagerBl().removeAdmin(sess, facility, group);
+			} catch (GroupNotAdminException e) {
+				throw new ConsistencyErrorException(e);
```

Comment:
Since we delete the group here, consistency in rights is not so relevant. It basically means, somebody removed relation during your delete operation. Please catch the exception and just log in as a WARN level instead of re-thorwing it as ConsistencyErrorException.